### PR TITLE
Feature/us 0017852

### PR DESCRIPTION
**Pull request by gaviriajesus**
This is a Test

### CODE REVIEW

**Comment by gaviriajesus** [comment]
This is a comment

**Comment by jesusgaviria** [comment]
Comment From jg

File: .gitignore [hunk 1]
```diff
@@ -54,3 +54,7 @@ objectTranslations/*
 
 sites/*
 siteDotComSites/*
+
+##IDEA Files To be Ignored
+.idea
```
Comment (jesusgaviria) [inline .gitignore]: This Gitignore?

**Comment by jesusgaviria** [comment]
Standard comment

**Comment by jesusgaviria** [comment]
I'm reopening with comment

**Pull request merged**